rest-endpoint-draft-readme.txt

```diff
@@ -0,0 +1,93 @@
+-- create a mapping:
```

Comment:
We would need to return resources based on the web services module's API, let's not create a deviant resource.